### ISSUE TITLE
Getting `Module Not Found` error for `plyer` library

### ISSUE THREAD

**Issue by OnlinePage** [Nuitka/Nuitka#2720]
Hi, i was compiling one of my python script to exe , my script has an import like below for showing windows notificaitons:
```python
from plyer import notification
#my rest codes goes here
```
Now after hitting the command like command , it was successfully built , however i am getting `Moudle not found error` for `plyer`. Below is the complete log:
```python
Nuitka-Options: Used command line options: --follow-imports --plugin-enable=upx --windows-icon-from-ico=.\myapp_icon.ico
Nuitka-Options: --output-dir=.\myAppBuild\myAppNuitka\ --enable-plugin=tk-inter --onefile --standalone .\myApp.py
Nuitka-Options: Following all imports is the default for onefile mode and need not be specified.
Nuitka: Starting Python compilation with Nuitka '2.0.3' on Python '3.10' commercial grade 'not installed'.
Nuitka-Plugins:options-nanny: Note, when using 'tkinter', consider using '--disable-console' option. Otherwise a terminal window will open.
Nuitka-Plugins:options-nanny: However for debugging, terminal output is the easiest way to see informative traceback and error information, so
Nuitka-Plugins:options-nanny: delay this until your program is working and remove once you find it non-working, and use '--enable-console' to
Nuitka-Plugins:options-nanny: make it explicit and not see this message.
Nuitka: Completed Python level compilation and optimization.            
Nuitka: Generating source code for C backend compiler.
Nuitka: Running data composer tool for optimal constant value handling.                       
Nuitka: Running C compilation via Scons.
Nuitka-Scons: Backend C compiler: cl (cl 14.3).
Nuitka-Scons: Backend linking program with 118 files (no progress information myappilable for this stage).
Nuitka-Scons: Compiled 118 C files using clcache with 118 cache hits and 0 cache misses.
Nuitka-Postprocessing: Adding 1 icon(s) from icon file '.\myapp_icon.ico'.
Nuitka-Plugins:data-files: Included data file 'certifi\cacert.pem' due to package data for 'certifi'.
Nuitka-Plugins:tk-inter: Included 87 data files due to Tk needed for tkinter usage.
Nuitka-Plugins:tk-inter: Included 836 data files due to Tcl needed for tkinter usage.
Nuitka-Postprocessing: Creating single file from dist folder, this may take a while.         
Nuitka-Onefile: Running bootstrap binary compilation via Scons.
Nuitka-Scons: Onefile C compiler: cl (cl 14.3).
Nuitka-Scons: Onefile linking program with 1 files (no progress information myappilable for this stage).
Nuitka-Scons: Compiled 1 C files using clcache with 1 cache hits and 0 cache misses.
Nuitka-Postprocessing: Adding 1 icon(s) from icon file '.\myapp_icon.ico'.
Nuitka-Onefile: Using compression for onefile payload.
Nuitka-Onefile: Onefile payload compression ratio (28.61%) size 31752033 to 9084390.                  
Nuitka-Onefile: Keeping onefile build directory 'myAppBuild\myAppNuitka\myApp.onefile-build'.
Nuitka: Keeping dist folder 'myAppBuild\myAppNuitka\myApp.dist' for inspection, no need to use it.
Nuitka: Keeping build directory 'myAppBuild\myAppNuitka\myApp.build'.
Nuitka-Plugins:upx: Compressing 'myAppBuild\myAppNuitka\myApp.exe'.
Nuitka: Successfully created 'myAppBuild\myAppNuitka\myApp.exe'.

```
i am not sure whats the current fix for it😨. Any help is highly appreciated!
For incase of pyinstaller it was easily handled via `--hidden-import plyer.platforms.win.notification` 

Below is my `Nuitka` details:
```python
Name: Nuitka
Version: 2.0.3
Summary: Python compiler with full language support and CPython compatibility
Home-page: https://nuitka.net
Author: Kay Hayen
Author-email: Kay.Hayen@gmail.com
License: Apache License, Version 2.0
Location: ....
Requires: ordered-set, zstandard
Required-by: 
```

**Comment by kayhayen** [comment]
In the template, we have made it clear, that not filling out the issue template is not acceptable. I am closing this kind of report as invalid. The main reason is that we do not want to be left guessing. For you it's easy to erase the issue template and expect us to ask you questions only as necessary, but we cannot do that. We would much rather wait for you to create a new report, fill out the template, or wait for somebody with the necessary skill to do it.

In short: No issue template, no fun. Therefore we are  likely just deleting the issue, as it serves as a bad example and detraction for searches.  And even worse, it might keep good reporters from reporting as is this "already known."

**Comment by kayhayen** [comment]
Sometimes I am really bad at reading, sorry.

**Comment by kayhayen** [comment]
Ok, should I be surprised that the minimal reproducer works for me.

**Comment by kayhayen** [comment]
Try again, with actual correct information, this was an unfortunate waste of time.
